### PR TITLE
Add Core Team Call Notes of 2022-09-02

### PR DESCRIPTION
**Pull request by lschuermann**
### Pull Request Overview

This pull request adds notes of the core team call on the 2nd of September, 2022.

### Testing Strategy

This pull request was tested by spellcheck / proofreading.

### TODO or Help Wanted

N/A

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [ ] Ran `make prepush`.

### CODE REVIEW

**Review by brghena — DISMISSED** [review]

**Review by jrvanwhy — DISMISSED** [review]
Do we want to correct the date in the PR title?

File: doc/wg/core/notes/core-notes-2022-09-02.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+# Tock Core Notes of 2022-09-02
+
+Attendees:
+- Brad Campbell
+- Chris Franzt
```
Comment (hudson-ayers) [inline doc/wg/core/notes/core-notes-2022-09-02.md:5]:
```suggestion
- Chris Frantz
```

**Review by hudson-ayers — APPROVED** [review]

File: doc/wg/core/notes/core-notes-2022-09-02.md (hunk 1 above)
Comment (lschuermann) [reply doc/wg/core/notes/core-notes-2022-09-02.md:5]: Argh, sorry!

**Comment by lschuermann** [comment]
> Do we want to correct the date in the PR title?

Yeah, we should always try to keep the PR title up to date (and correct in the first place), as that will also be included in the merge commit. Seems like I forgot to update the date in this PR virtually everywhere, finger's crossed the notes are actually correct though.

**Pull request merged**